plugins/content-manager/src/main/java/com/wazuh/contentmanager/action/cti/GetChangesAction.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2024, Wazuh Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.wazuh.contentmanager.action.cti;
+
+import org.apache.hc.client5.http.async.methods.SimpleHttpResponse;
+import org.apache.hc.core5.http.Header;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.core.rest.RestStatus;
+import org.opensearch.core.xcontent.*;
+import org.opensearch.rest.BytesRestResponse;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.wazuh.contentmanager.ContentManagerPlugin;
+import com.wazuh.contentmanager.model.ctiapi.Offsets;
+import com.wazuh.contentmanager.privileged.PrivilegedHttpAction;
+
+/**
+ * Action class handling Offsets logic. This is used to get the json patches to the current
+ * vulnerability data
+ */
+public class GetChangesAction {
+
+    private static String FROM_OFFSET_FIELD = "from_offset";
+    private static String TO_OFFSET_FIELD = "to_offset";
+    private static String WITH_EMPTIES_FIELD = "with_empties";
+    private String fromOffset = null;
+    private String toOffset = null;
+    private String withEmpties = null;
```

Comment:
Do these vars need to be initiated? Wouldn't it be the same if we remove the `= null`
```suggestion
    private String fromOffset;
    private String toOffset;
    private String withEmpties;
```